pycytominer/tests/test_cyto_utils/test_load.py

```diff
@@ -188,11 +215,35 @@ def test_load_npz():
         IndexError, match="OutOfBounds indexing via location_x_col_index"
     ):
         load_npz_locations(
-            example_npz_file_locations, location_x_col_index=2, location_y_col_index=1
+            example_npz_file_locations,
+            location_x_col_index=2,
+            location_y_col_index=1,
         )
     with pytest.raises(
         IndexError, match="OutOfBounds indexing via location_y_col_index"
     ):
         load_npz_locations(
-            example_npz_file_locations, location_x_col_index=0, location_y_col_index=2
+            example_npz_file_locations,
+            location_x_col_index=0,
+            location_y_col_index=2,
         )
+
+
+def test_is_path_a_parquet_file():
+    # checking parquet file
+    check_pass = is_path_a_parquet_file(output_data_parquet)
+    check_fail = is_path_a_parquet_file(output_data_file)
+
+    # checking if the correct booleans are returned
+    assert (check_pass, True)
+    assert (check_fail, False)
+
+    # loading in pandas dataframe from parquet file
+    parquet_df = pd.read_parquet(output_data_parquet)
+    parquet_profile_test = load_profiles(output_data_parquet)
+    pd.testing.assert_frame_equal(parquet_profile_test, parquet_df)
```

Comment:
I asked this in my previous review, but maybe I was not clear.

You're testing parquet vs. parquet and csv vs. csv (which is great!) but what about parquet vs. csv? 

Maybe at the very end of this test, add the following:

```python
pd.testing.assert_frame_equal(parquet_profile_test, csv_profile_test)
```